templates/default/less/variables.less

```diff
@@ -189,6 +199,10 @@
 @dropdown-link-hover-bg: #f2f2f2;
 @dropdown-link-hover-bg: lighten(@brand-primary, 53%);;
 
+//== determines display of icons in dropdowns
+//== 'none' for no display and 'inline' for display
```

Comment:
Use to Bootstrap convention to describe variables. Try to stick with short description for variables.